apps/emqx_bridge_kinesis/src/emqx_bridge_kinesis.erl

```diff
@@ -28,6 +34,37 @@ namespace() ->
 roots() ->
     [].
 
+fields(Field) when
+    Field == "get_connector";
+    Field == "put_connector";
+    Field == "post_connector"
+->
+    emqx_connector_schema:api_fields(
+        Field,
+        ?CONNECTOR_TYPE,
+        connector_config_fields()
+    );
+fields(action) ->
+    {?ACTION_TYPE,
+        hoconsc:mk(
+            hoconsc:map(name, hoconsc:ref(?MODULE, kinesis_action)),
+            #{
+                desc => <<"Kinesis Action Config">>,
+                required => false
+            }
+        )};
+fields(action_parameters) ->
+    fields(producer);
+fields(kinesis_action) ->
+    emqx_bridge_v2_schema:make_producer_action_schema(
+        hoconsc:mk(
+            hoconsc:ref(?MODULE, action_parameters),
```

Comment:
It will introduce a duplicated field `local_topic` for actions creating. e.g:
<img width="1325" alt="image" src="https://github.com/emqx/emqx/assets/13825269/6483574c-e421-4835-8e28-73532f8fad5c">